solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -109,36 +180,38 @@ contract RandomBeacon is Ownable {
         );
     }
 
-    /// @notice Updates the values of group creation parameters
+    /// @notice Updates the values of group creation parameters.
     /// @dev Can be called only by the contract owner, which should be the
-    ///      random beacon governance contract
+    ///      random beacon governance contract. The caller is responsible for
+    ///      validating parameters.
     /// @param _groupCreationFrequency New group creation frequency
     /// @param _groupLifetime New group lifetime
     /// @param _dkgResultChallengePeriodLength New DKG result challenge period
     ///        length
-    /// @param _dkgSubmissionEligibilityDelay New DKG submission eligibility
-    ///        delay
+    /// @param _dkgResultSubmissionEligibilityDelay New DKG submission
```

Comment:
"New DKG submission" => "New DKG result submission"

Reply:
I think this one is still missing.

Reply:
Ops, my bad. Please see https://github.com/keep-network/keep-core/pull/2638/commits/bce451f4adca8b55455640f93d918679a107d783.